Packages/vcs/Lib/colormap.py

```diff
@@ -231,12 +288,12 @@ def __init__(self, Cp_name, Cp_name_src='default'):
         # Note: See RGB_Table Class for "index" setting of the colormap entries
 
     # Set a colorcell RGB
-    def setcolorcell(self, index, red, green, blue):
+    def setcolorcell(self, index, red, green, blue, alpha=100.):
```

Comment:
Legacy support. All of the old usages of this function would have generated opaque colors, no reason to change that behavior.